### DISCUSSION TITLE
How do I Synchronize Multiple Vaults?

### DISCUSSION THREAD

**Discussion by uninvitedguest** [vrtmrz/obsidian-livesync#328] [Q&A]
I successfully set up the synchronization of my first vault - fantastic stuff! My issue now is in synchronizing additional vaults.

1. I create a new vault on my desktop.
2. I install the LiveSync plugin to this vault. 
3. I set up the plugin using a different database name.
4. I use the URI to set up my mobile devise as a secondary device.
5. Eventually, the contents of my first vault are synchronized in to this brand new vault that should be using a different DB. :(

**Comment by dlphnkck** [comment]
Hi. I am not familiar with this issue.
It's not like you mistakenly copied the set up URI from the first vault, is it?
Would the following topic be helpful?

[Sync multiple vaults · vrtmrz/obsidian\-livesync · Discussion \#141](https://github.com/vrtmrz/obsidian-livesync/discussions/141)

I wish you good luck.
